tasks/libs/junit_upload.py

```diff
@@ -246,3 +250,32 @@ def repack_macos_junit_tar(infile, outfile):
         # pack all files to a new tarball
         for f in os.listdir(tempd):
             outfp.add(os.path.join(tempd, f), arcname=f)
+
+
+def retrieve_jira_card(xmlfile_path):
+    """
+    Search in jira if a card already exist for the given testsuite
+    """
+    from .pipeline_notifications import send_slack_message
+    from jira import JIRA
+
+    tree = ET.parse(xmlfile_path)
+    auth = ("mail", "token")
```

Comment:
pending creation of a service account (see https://datadog.freshservice.com/support/tickets/372220)